diff -up qemu-8.1.0-rc3/tcg/i386/tcg-target.c.inc.2~ qemu-8.1.0-rc3/tcg/i386/tcg-target.c.inc
--- qemu-8.1.0-rc3/tcg/i386/tcg-target.c.inc.2~	2023-08-14 18:30:42.883408844 +0200
+++ qemu-8.1.0-rc3/tcg/i386/tcg-target.c.inc	2023-08-14 18:31:35.542918505 +0200
@@ -595,6 +595,10 @@ static void tcg_out_vex_opc(TCGContext *
 {
     int tmp;
 
+    if (opc & P_GS) {
+        tcg_out8(s, 0x65);
+    }
+
     /* Use the two byte form if possible, which cannot encode
        VEX.W, VEX.B, VEX.X, or an m-mmmm field other than P_EXT.  */
     if ((opc & (P_EXT | P_EXT38 | P_EXT3A | P_VEXW)) == P_EXT
